.github/ISSUE_TEMPLATE/1_bug_report_form.yml

```diff
@@ -17,6 +17,12 @@ body:
       description: A concise description of what issue you're experiencing. **Example:** "When I click on the 'Sign in' button, I expect to be redirected to the login page, but instead I get a 404 error."
     validations:
       required: true
+  - type: input
+    attributes:
+      label: URL of the page where the Issue is observed.
+      description: Kindly provide the URL of the specific page where you are experiencing the issue.
```

Comment:
The URL of the ...

(This is to be consistent with the other descriptions, which don't say "Kindly" or equivalent.)

Reply:
Done